test/plenario_web/controllers/web/page_controller_test.exs

```diff
@@ -177,4 +197,10 @@ defmodule PlenarioWeb.Web.Testing.PageControllerTest do
     conn = get(conn, page_path(conn, :explorer), %{"starting_on" => "woopwoop", "ending_on" => "2000-01-01"})
     assert get_flash(conn)["error"] =~ "Invalid date woopwoop"
   end
+
+   @tag :anon
+  test "explorer receives a terribly fake date", %{conn: conn} do
+    conn = get(conn, page_path(conn, :explorer), %{"starting_on" => "2000-01-01", "ending_on" => "2000-19-11"})
```

Comment:
Ahh yes, the fabled nineteenth month AKA Septendecember

Reply:
beware the ides of septendecember